sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -382,7 +382,7 @@ public synchronized void onActivityCreated(
 
     firstActivityCreated = true;
 
-    if (fullyDisplayedReporter != null) {
+    if (performanceEnabled && ttfdSpan != null && fullyDisplayedReporter != null) {
```

Comment:
I've added this, as otherwise we'd add a new listener inside FullyDisplayedReporter without ever clearing that list.